packages/aws/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.30.2"
+  changes:
+    - description: Added the support for listeners with ALPN policy extension in ELB dataset for Network Load Balancers.
```

```suggestion
    - description: Add the support for listeners with ALPN policy extension in ELB dataset for Network Load Balancers.
```


To match the tone with other entries in this file.

---

Fixed. Marking as resolved.